src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java

```diff
@@ -100,7 +100,12 @@ public String lti3(HttpServletRequest req, HttpServletResponse res, Model model)
         String link = req.getParameter("link");
         String altDomain = req.getHeader("x-formated-host");
         if (StringUtils.isNotBlank(altDomain)){
-            altDomain = DomainUtils.extractDomain(altDomain);
+            String extractedDomain = DomainUtils.extractDomain(altDomain);
+            if (extractedDomain == null && DomainUtils.isWildcardDomain(altDomain, ltiDataService.getLocalUrl())) {
```

Comment:
@ddelblanco Is altDomain what you would expect here?

Reply:
now, with the last changes,  it looks good to me

Reply:
I just want to confirm that we do not need to worry about checking the domain.url here?